apps/api/src/examples/extract-data-attributes.ts

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
Not necessary, this should be turned into brief documentation in the `firecrawl-docs` repo